content/news/2023-07-09-bevy-0.11/index.md

```diff
@@ -290,11 +292,13 @@ Since normal maps do not "move" the shaded areas, merely shade them
 differently, we get those awkward specular highlights. With parallax mapping,
 they are gone.
 
-<b style="display:block; margin-bottom: -18px">Drag this image to compare</b>
-<div class="image-compare" style="aspect-ratio: 16 / 9; --slider-value: 39%" data-title-a="Normals Only" data-title-b="Parallax+Normals">
-  <img class="image-a" alt="Normal Mapping Only" src="parallax_mapping_normals.jpg">
-  <img class="image-b" alt="Parallax & Normal Mapping" src="parallax_mapping_depth.jpg">
-</div>
+{{ compare_slider(
+    before_title="Normals Only",
+    before_image="parallax_mapping_normals.jpg",
+    after_title="Parallax & Normal Mapping",
+    after_image="parallax_mapping_depth.jpg",
+    start_slider_at="40%"
+) }}
```

Comment:
These titles should match the `data-title-*` attributes rather than the alt-text; cart shortened those attributes so that the image titles would visually fit on small displays (phones, &c).

Reply:
oh maybe short-codes can't include optional arguments? that's a shame..

Reply:
short codes support optional arguments. the `rust_type` short code makes use of this regularly

Reply:
I'm worried about the complexity of the macro. It might intimidate potential contributors into glazing over the shortcode. But it's a relatively minor concern compared to more detailed alts